### ISSUE TITLE
Acessibility improvements

### ISSUE THREAD

**Issue by DoviMaj** [TheOdinProject/theodinproject#1682]
Lighthouse points to those improvements on several pages of the site: 

 -  [ ] Background and foreground colors do not have a sufficient contrast ratio.
 -  [x] Image elements do not have [alt] attributes. 
 -  [ ] Links do not have a discernible name. 
 -  [ ] [aria-hidden="true"] elements contain focusable descendants.
 -  [ ] Lists do not contain only `<li>` elements and script supporting elements (`<script> `and `<template>`).

**Comment by KevinMulhern** [comment]
Hey @DoviMaj, I'm doing a bit of issue clean up this evening. Would there be anyway to make this one more actionable?

**Comment by DoviMaj** [comment]
**Background and foreground colors do not have a sufficient contrast ratio:**
Would need some redesign since it follows the logo colors. it fails on a lot of items on the website, we were waiting for Ada's input I think.

The rest of them would involve figuring out what the error is and fixing them, I'm open to suggestions on how to make this issue more actionable. @KevinMulhern

**Comment by KevinMulhern** [comment]
@adachiu can we get your input on the contrast issue for this please?

@DoviMaj did lighthouse flag a lot of issues for the others? if there aren't too many it would be worth listing them out in their own issue. Otherwise we could make separate issues for them with instructions on how to see them with lighthouse. What do you think?

**Comment by DoviMaj** [comment]
@KevinMulhern there weren't a lot of flags for the other issues, the contrast one was by far the most flagged one. 

Should I open an issue for each one of them?

**Comment by ZachPoole** [comment]
Hey just an issue I found with some contrast that sounds related. 
The 404 page needs more contrast for the text. 
For this specific page I went to https://www.theodinproject.com/dashboard

![image](https://user-images.githubusercontent.com/46611063/147893001-8fe15910-79ed-4b65-99d8-827fa4383e99.png)

**Comment by ChargrilledChook** [comment]
Probably be good to have a look at again once Tailwind conversion is complete, although perhaps most of this work could be rolled into #2893

**Comment by Asartea** [comment]
I think with all of the recent work done this should be all done? Random sampling with Lighthouse (run via unlighthouse; sample size = 27, including all static pages) suggests that the only remaining outstanding issues are a few heading issues, which are being done in #3965

**Comment by ChargrilledChook** [comment]
Sounds good to me 🐢
